aocd/models.py

```diff
@@ -13,28 +11,24 @@
 import sys
 import time
 import webbrowser
-from datetime import datetime
-from datetime import timedelta
+from datetime import datetime, timedelta
 from textwrap import dedent
 
 import bs4
 import pkg_resources
 import requests
-from termcolor import colored
-from termcolor import cprint
-
-from .exceptions import AocdError
-from .exceptions import DeadTokenError
-from .exceptions import UnknownUserError
-from .exceptions import PuzzleUnsolvedError
-from .exceptions import PuzzleLockedError
-from .utils import AOC_TZ
-from .utils import _ensure_intermediate_dirs
-from .utils import atomic_write_file
-from .utils import get_owner
+from numpy import complexfloating as np_complex
```

Comment:
You need to add `numpy` as a dependency in `install_requires` in `setup.py`.

Reply:
It's in there: https://github.com/wimglenn/advent-of-code-data/pull/104/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R45

Reply:
Ah, sorry, I missed that.